locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml

```diff
@@ -35,41 +35,41 @@ de:
           s: 6
           from: 00:00:35,20
           to: 00:00:42,19
-          text: My parents, my father was a metal worker, a hard working man. He repaired
-            ships.
+          text: Meine Eltern … mein Vater war ein Handwerker, ein Metallarbeiter,
```

Comment:
I'm afraid as opposed to English, in German there needs to be a space before an ellipsis. Unless the word itself is cut off: So "Meine Eltern …" vs. "Mein Elt…" ;)